Assets/Dev/Exostin/ExostinScripts/TitleScreenHandler.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class TitleScreenHandler : MonoBehaviour
+{
+    public void PlayButton()
+    {
+        // You need to put the number of the main scene from build settings in here
+        SceneManager.LoadScene(1);
```

Comment:
Could we use bracket notation instead here? `SceneManager.LoadScene(["GathdarScene"])` or something similar. I don't know how much of the path would need to be included here, but it would be clearer to other devs why we end up in that scene (in case they don't know about BuildSettings)

Reply:
Sure, I wrote it this way just because it is said to be the better practice performance-wise (engine doesn't have to search by name, but just choose an index)

Reply:
Hmm, but if we do it by index we won't have to change the name in script everytime the scene is changed so now I would write "GathdarDev" and then it would need to be changed to for example ArthurScene once we get the PR merged from him

Reply:
We decided to keep it that way and I'll just show how Build Settings work in the nearest demo meeting, resolving